src/components/entity/ha-state-label-badge.js

```diff
@@ -39,6 +39,8 @@ export default new Polymer({
 
   computeClasses(state) {
     switch (state.domain) {
+    case 'binary_sensor':
+      return state.state === 'on' ? 'green' : 'red';
```

Comment:
Can we use the same logic as we currently do for `script`? So on = blue, off is grey. That way we are consistent to indicate to the user that a badge is 'on'